fulp_modules/features/spells/_spells.dm

```diff
@@ -256,7 +256,7 @@
 /obj/item/gun/magic/wand/directable_cat_meteor/Initialize(mapload)
 	. = ..()
 	AddComponent(/datum/component/scope/magic)
-	register_context()
+	register_item_context()
```

Comment:
What is this context doing? Previously it looks like you used it to tell the user that you can RMB to toggle the scope (which should be done on the component btw, not here), but I don't see any new context for this change to make sense?

Reply:
The change goes from `register_context()` to `register_item_context()` because it's an item that the context is being registered to. I don't think it worked previously with `register_context()`, but I recall it working during debugging when `register_item_context()` was used instead.

Also, I do agree that adding the context via the component would be ideal, but I'm not sure how to go about doing that (nor can I find examples of it being done elsewhere in the codebase upon a quick glance at a few other item components).

Reply:
register_context is for when you want context when you hover over that item, register_item_context is for when you want context when you're hovering over something else with this item in your hand, from what I know.

For components that do it,
``customizable_reagent_holder``
``food_storage``
``lockable_storage``
``profound_fisher``
redirect_attack_hand_from_turf``
``rotation``
``toggle_suit``
``material_container``
``can_barricade``
``processable``
among others

Reply:
I see, thank you very much for the examples! :D 

I have adapted what was done with `customizable_reagent_holder` and applied it to the magic scope component. I did change the adapted code to use the 'item_context' variation of screentip code since we want the screentip to originate from the item/gun/wand itself hovering over something— right?

Reply:
The screen tip is right clicking on the wand itself so yeah it's context on the item, not when hovering over something else.

Reply:
Line removed.